db_mutex/__init__.py

```diff
@@ -1,5 +1,4 @@
 # flake8: noqa
-from .db_mutex import db_mutex
```

Comment:
@wesokes any reason this was removed? If the new API doesn't support

```
from db_mutex import db_mutex
```

then you will want to update the doc examples. I was following the doc examples and it errored :/ Let me know if you want me to revert back to the old API or fix the docs. Either is fine